src/frontend/web_application/src/scenes/ContactBook/components/ContactList/index.jsx

```diff
@@ -65,10 +76,10 @@ const ContactList = ({ contacts, sortView }) => {
 
 ContactList.propTypes = {
   contacts: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
-  sortView: PropTypes.string,
+  sortView: PropTypes.oneOf([SORT_VIEW_GIVEN_NAME, SORT_VIEW_FAMILY_NAME]).isRequired,
```

Comment:
this give an error:
Failed prop type: Invalid prop `sortView` of value `given_name` supplied to `ContactList`, expected one of [null,null].
despite thoses variables are imported and used in function... what's the problem?